DOCKER/README.md

```diff
@@ -14,26 +14,26 @@
 - `0.8.0`, `0.8` [(Dockerfile)](https://github.com/tendermint/tendermint/blob/bf64dd21fdb193e54d8addaaaa2ecf7ac371de8c/DOCKER/Dockerfile)
 - `develop` [(Dockerfile)](https://github.com/tendermint/tendermint/blob/master/DOCKER/Dockerfile.develop)
 
-`develop` tag points to the [develop](https://github.com/tendermint/tendermint/tree/develop) branch.
+`develop` tag points to the [develop](https://github.com/tendermint/tendermint/tree/master) branch.
```

Comment:
we should remove both the tag and this sentence

Reply:
is docker file being built from master every push now?

in dockerfile.develop it might be good to change to master if this is not the case

Reply:
> is docker file being built from master every push now?

no

> in dockerfile.develop it might be good to change to master if this is not the case

maybe we should just delete it. I don't use it anymore